api/e_appendix.txt

```diff
@@ -3,7 +3,11 @@
 // http://creativecommons.org/licenses/by/4.0/
 
 [appendix]
-= Changes
+[[changes_to_opencl]]
+= Changes to OpenCL
+
+The following sections describe changes to the OpenCL API and OpenCL C which
+were made in each successive version.
```

Comment:
This is phrased awkwardly.  Consider something like: "changes to the OpenCL API and OpenCL C Language specifications, which were made..."

Reply:
Should be fixed - I added a bit of verbosity in the subsection headers to make it more clear where the changes applied.